omise/payment_methods/true_money.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . 'omise/payment_methods/_offsite.php';
+}
+
+class OmisePaymentMethod_TrueMoney extends OmiseOffsitePaymentMethod
+{
+
+    const
+        NAME = 'TrueMoney',
+        PAYMENT_OPTION_NAME = 'omise-truemoney-payment',
+        DEFAULT_TITLE = 'True Money',
+        TEMPLATE = 'truemoney_payment',
+        SWITCH_DESCRIPTION = 'Enables payments by True Money (currently only available in Thailand).'
+    ;
+
+
+    public static
+        $usedSettings = array('true_money_status'),
```

Comment:
maybe better to be `truemoney_status`? (just to make it consist with its `truemoney_*` payment code.

Reply:
ok - changed

Reply:
Reverted - this name has to be snake cased version of name